pkg/ccl/backupccl/backup_planning.go

```diff
@@ -567,6 +567,7 @@ func backupPlanHook(
 
 		var tables []catalog.TableDescriptor
 		statsFiles := make(map[descpb.ID]string)
+		// TODO(pbardea): Let's check the privs for UDTs and UDSs here.
```

Comment:
the privileges are set up for these now, so there isn't a blocker anymore here

Reply:
Ack - planning on doing it in a separate PR for easier history tracking.